molgenis-model-registry/src/main/resources/templates/view-standardsregistry_docs.ftl

```diff
@@ -6,15 +6,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-<#-- FIXME enable resource_href>
-<#--<link rel="icon" href="<@resource_href "/img/molgenis.ico"/>" type="image/x-icon">-->
-<#--<link rel="stylesheet" href="<@resource_href "/css/bootstrap.min.css"/>" type="text/css">-->
-    <link rel="icon" href="/img/molgenis.ico" type="image/x-icon">
+    <#-- FIXME enable resource_href>
+    <#--<link rel="icon" href="<@resource_href "/img/molgenis.ico"/>" type="image/x-icon">-->
+    <#--<link rel="stylesheet" href="<@resource_href "/css/bootstrap.min.css"/>" type="text/css">-->
+    <link rel="icon" href="<@resource_href "/img/favicon.ico"/>" type="image/x-icon">
```

Comment:
I seriously doubt that this call of resource_href suddenly will work if the other's didn't. Shall we ditch it altogether here?